brian2/tests/test_stateupdaters.py

```diff
@@ -754,7 +766,7 @@ def test_check_for_invalid_values_linear_integrator():
                 assert 'invalid_values' in repr(clog)
             else:
                 assert G.x[0] != 0
-        except UnsupportedEquationsException:
+        except BrianObjectException:
```

Comment:
Should this be made more specific or unnecessary do you think?

Reply:
I added an assertion which checks the "cause" of the ecxeption.